hack/delete.sh

```diff
@@ -55,6 +48,12 @@ if [ "$all" = "true" ]; then
   with_openFuncAsync=true
 fi
 
+if [ "$poor_network" = "false" ]; then
```

Comment:
Currently, the installation of `cert-manager` is placed in `bundle.yaml`, so the delete script should not delete `cert-manager`

Reply:
done